bin/mmdblookup.c

```diff
@@ -18,7 +18,7 @@
 #include <unistd.h>
 #endif
 
-#define LOCAL
+#define LOCAL static
```

Comment:
I wonder if we should just get rid of `LOCAL` everywhere. I find it more confusing than helpful.

Reply:
I'm on board with that. Using `static` seems clear to me. Not sure if `LOCAL` has any advantages.